src/vex.js

```diff
@@ -0,0 +1,287 @@
+// String to DOM function
+var domify = require('domify')
+// classList polyfill for old browsers
+require('classlist-polyfill')
+
+// Object.assign polyfill
```

Comment:
Since `browserify` is used to bundle this project together now, would it make sense to break these functions/polyfills/utils into separate modules e.g. `src/object-assign.js`?